packages/db-schema/src/voting/defs.ts

```diff
@@ -14,22 +12,11 @@ import {
   uuid,
 } from "drizzle-orm/pg-core";
 
-import { competitions } from "../core/defs.js";
-import { blockchainAddress, tokenAmount } from "../util.js";
-
-const bytea = customType<{
-  data: Uint8Array | Buffer; // what your app uses
-  driverData: Buffer; // what node-postgres returns
-  notNull: false;
-  default: false;
-}>({
-  dataType: () => "bytea",
-  toDriver: (v) => (v instanceof Buffer ? v : Buffer.from(v)),
-  fromDriver: (v) => v, // Buffer
-});
+import { agents, competitions, users } from "../core/defs.js";
+import { blockchainAddress, bytea, tokenAmount } from "../util.js";
 
 export const epochs = pgTable("epochs", {
-  id: uuid().primaryKey().notNull(),
+  id: uuid().primaryKey().notNull().defaultRandom(),
```

Comment:
Saw you can set a default uuid in postgres. Makes it nice not having to provide it in code.